platformio.ini

```
@@ -131,7 +131,7 @@ extra_scripts = post:esphome/components/esp32/post_build.py.script
 ; This are common settings for the ESP32 (all variants) using IDF.
 [common:esp32-idf]
 extends = common:idf
-platform = platformio/espressif32 @ 3.5.0
+platform = platformio/espressif32 @ 4.4.0
 platform_packages =
     platformio/framework-espidf @ ~3.40302.0
```

```suggestion
    platformio/framework-espidf @ ~3.40401.0
```

Ah sorry this one needs to be changed as well